frontends/web/src/components/sidebar/sidebar.tsx

```diff
@@ -158,14 +168,15 @@ class Sidebar extends Component<Props> {
   };
 
   public render() {
+    const { keystores } = this.state;
     const {
       t,
       deviceIDs,
       accounts,
-      keystores,
       activeSidebar,
       sidebarStatus,
     } = this.props;
+    console.log('deviceIDs/keystores/accounts', deviceIDs, keystores, accounts);
```

Comment:
when an error occurs, keystores is empty, but accounts still has the accounts

Reply:
![Screen Shot 2023-10-24 at 09 16 42](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/63a8ff5a-bc56-49a8-9bcb-b0a96fff0bb7)

Reply:
It seems to me that the notification order from the backend is 1. keystore 2. accounts 3. device.
See in particular https://github.com/digitalbitbox/bitbox-wallet-app/blob/master/backend/backend.go#L552-L555 that emits the keystore event before the account one.

Reply:
Opened a separate PR to fix #2311

Reply:
thank you ❤️ 

I'll drop the last commit in this PR

Reply:
since this PR error is not related to this PR I'll update the last commit message, and think we could merge this anyway. WDYT?

Reply:
Rebased onto staging-watchonly here: https://github.com/digitalbitbox/bitbox-wallet-app/pull/2389

I wonder if the error when unpluging a device  still occurs

cc @benma just FYI.